grabserial: add package

vmandela opened the pull request:
This is the grabserial python utility for timestamping serial port output. 

Homepage: https://github.com/tbird20d/grabserial

Please let me know if the package location in nixpkgs needs to be changed or any other change needs to be made.

Thanks and Regards,
Venkat

vmandela committed:
grabserial: add package

pSub commented on pkgs/development/tools/grabserial/default.nix:
```diff
@@ -0,0 +1,21 @@
+  buildInputs = [ pythonPackages.pyserial ];
+
+  meta = {
+    description = "python-based serial dump and timing program"
```
Here is a semicolon missing and `description`s should start with a capital letter.

bjornfor commented on pkgs/development/tools/grabserial/default.nix:
```diff
@@ -0,0 +1,21 @@
+
+buildPythonPackage rec {
+
+ name = "grabserial-8b9c98ea";
```
A git commit id is not a good version number (there is no meaningful way to compare two hashes). If a package has no release number, we tend to use the date of the commit instead (YYYYMMDD).

vmandela commented:
@pSub I fixed the syntax errors in the meta information. I should have caught that before submitting the pull request.

@bjornfor I have updated the version number string to the last commit date.

Thanks and Regards,
Venkat

vmandela commented:
The Travis CI build failed because of a build issue unrelated to grabserial. Can anyone please help me resolve this issue?

Thanks and Regards,
Venkat

==> Merging PR into base
error: cannot import ‘/nix/store/zapaadgbk6pbfbxdiq1pk6ax62dra3b0-nixui/default.nix’, since path ‘/nix/store/01rsibv0w4h8yhcm8fv0qfv0p3sw655v-nixui.drv’ is not valid, at "/home/travis/.nox/nixpkgs/pkgs/tools/package-management/nixui/default.nix":13:12

bjornfor commented:
@vmandela: Don't worry about Travis CI, we're used to it fail a lot :-)

Btw, I see that grabserial-1.6.0 was released 2014-01-07 (changelog in grabserial script). What's the reason for going with git HEAD instead of the release?

bjornfor commented:
Ah, upstream doesn't do releases it seems (no tags or archives...).

bjornfor commented:
Ooops, forgot we're in 2015! Never mind previous comment :-)

bjornfor commented:
Pushed to master with small fix (01fc3e0525c6c6c20cbd0b8f40e9a939505a5e36). Thanks!
